src/song.cpp

```diff
@@ -160,6 +167,13 @@ void		Song::init()
 		year = "";
 	}
 
+	/* original year from original date */
```

Comment:
I don't think so. I imagine most people who want to do something with this field would use a combination of both, rather than simply preferring one over the other.